pytorch_lightning/profiler/pytorch.py

```diff
@@ -253,16 +217,16 @@ def start(self, action_name: str) -> None:
 
         self.context_names[action_name] = "/".join(self.running_stack)
 
-        self._start(action_name)
+        self._start()
 
-    def _start(self, action_name: str) -> None:
+    def _start(self) -> None:
         if self.emit_nvtx:
-            self._create_profiler(action_name, torch.cuda.profiler.profile, enter=False)
-            self._create_profiler(action_name, torch.autograd.profiler.emit_nvtx)
+            self._create_profiler(torch.cuda.profiler.profile, enter=False)
+            self._create_profiler(torch.autograd.profiler.emit_nvtx)
         else:
-            self._create_profiler(action_name, torch.autograd.profiler.profile)
+            self._create_profiler(torch.autograd.profiler.profile)
 
-    def _create_profiler(self, action_name, profiler, enter=True):
+    def _create_profiler(self, profiler, enter=True):
```

Comment:
`action_name` was unused so removed it. Maybe it should be used?

Reply:
Fine :)